basics/readme.md

```diff
@@ -0,0 +1,11 @@
+## Lab1
+
+# Pluralize fuction
+This function Pluralize animals if its qty 0 or not 1
+
+# Coin Flip
+Function is flipping coins and records how many flips happened
+
+# Clock function
+outputs real time in the console until user stops it
+used as reference https://crunchify.com/how-to-run-a-program-forever-in-java-keep-running-main-thread-continuously/
```

Comment:
Thanks for specifying the resource you used! IMO, it's even more useful to include as a comment right near the code that you wrote using that resource; that way, if you come back to look at this code later, you can see exactly what resource you used to write this code. But this is certainly sufficient :)